pkg/graph/schema.graphql

```diff
@@ -1871,7 +1871,7 @@ TRBRequestChanges represents the possible changes you can make to a TRB request
 Fields explicitly set with NULL will be unset, and omitted fields will be left unchanged.
 https://gqlgen.com/reference/changesets/
 """
-input TRBRequestChanges @goModel(model: "map[string]interface{}") {
+input TRBRequestChanges @goModel(model: "map[string]interface{}") { # no change, i think
```

Comment:
Can probs just nix this comment once you're all done. I didn't see any necessary changes to this either, FWIW!

Reply:
you are right! sometimes i leave little notes to myself 😄 nice catch, thanks!

Reply:
It's actually helpful in review so that it shows up as something to review, to!